thunder/tests/test_grad.py

```diff
@@ -36,6 +36,8 @@
     "embedding",
     "index_put",
     "batch_norm",
+    # Requires CUDA runtime to be available (fails on CPU only runtimes).
+    "cuda",
```

Comment:
I suppose one fix would be to test if cuda is available and only if that's true add the string to this set? @IvanYashchuk